mlir/include/mlir/Dialect/Tosa/Utils/ConversionUtils.h

```diff
@@ -79,6 +81,141 @@ checkHasDynamicBatchDims(PatternRewriter &rewriter, Op op,
 LogicalResult EqualizeRanks(PatternRewriter &rewriter, Location loc,
                             Value &input1, Value &input2);
 
+LogicalResult EqualizeRanks(ImplicitLocOpBuilder &builder, Value &input1,
+                            Value &input2);
+
+namespace {
+
+// Creates a TOSA operation and performs shape inference on the individual
+// op. This allows shape inference during the TFLite to TOSA lowering.
```

Comment:
Better to rephrase this "This allows shape inference when lowering down to TOSA" instead of mentioning explicit framework names?

Reply:
done